### DISCUSSION TITLE
Render markdown without page breaks?

### DISCUSSION THREAD

**Discussion by amir734jj** [marp-team/marp#196] [Q&A] [answered]
I am using the core package to render HTML and I am wondering is it possible to **not** include page breaks? in my markdown file I use "---" for page breaks when converting to PDF, but I don't want page breaks when converting to HTML.

```javascript
var marp = new Marp({});

var { html, css } = marp.render("<markdown here>");
```

![image](https://user-images.githubusercontent.com/4202918/138614365-73d7fb59-d4b4-4003-a872-f2a0baa1fc60.png)

**Comment by yhatt** [comment, marked as answer]
Marp Core, extended from Marpit, has included styles for page breaks by [`printable` constructor option](https://marpit-api.marp.app/marpit).
https://github.com/marp-team/marpit/blob/01f564d8e08406a9b072af226943479c0c2f5cc8/src/postcss/printable.js

Set `printable: false` if you don't want to split pages while printing.

```javascript
new Marp({ printable: false })
```

Marp is designed for making presentation slides by splitting Markdown contents. If required rendering HTML into one page, you should use one of common Markdown rendering libraries instead of Marp. e.g. [markdown-it](https://github.com/markdown-it/markdown-it), [remark](https://github.com/remarkjs/remark), [marked](https://github.com/markedjs/marked)

**Reply by amir734jj** [reply]
Thank you so much. It worked!
